language/compiler/ir-to-bytecode/src/compiler.rs

```diff
@@ -414,7 +414,7 @@ pub fn compile_script<'a, T: 'a + ModuleAccess>(
     let parameters_sig_idx = context.signature_index(Signature(sig.parameters))?;
 
     record_src_loc!(function_decl: context, function.loc, 0);
-    let code = compile_function_body_impl(&mut context, function.value)?;
+    let code = compile_function_body_impl(&mut context, function.value)?.unwrap();
```

Comment:
maybe using expect with a message instead of `unwrap` could be better? 
we are going to do few passes to make those cleanups where needed but I mention in case you wanted to do that here and to make sure we change things as needed